lib/ClangImporter/ImportType.cpp

```diff
@@ -1926,7 +1926,8 @@ ParameterList *ClangImporter::Implementation::importFunctionParameterList(
     } else {
       if (auto refType = dyn_cast<clang::ReferenceType>(paramTy)) {
         paramTy = refType->getPointeeType();
-        isInOut = true;
+        paramSpecifier = paramTy.isConstQualified() ? ParamSpecifier::Shared
```

Comment:
"__shared" doesn't mean "shared value" is really means "non owning" (as Andrew helpfully reminded me). No matter what, It's still possible that copies will be made, since we don't support proper borrowing yet. But I'm told that's soon to come. (I think that's the same case for inout if the underlying object is constructed in Swift, so it's a bit of a mute point, though.)

Anyway, we shouldn't use "__shared" here. Let's just use the default parameter specifier and update it whenever we have something better. 

CC @atrick